homeassistant/components/roku/config_flow.py

```diff
@@ -119,11 +118,8 @@ def _show_form(self, errors: Optional[Dict] = None) -> Dict[str, Any]:
             try:
                 await self.hass.async_add_executor_job(validate_input, user_input)
                 return self.async_create_entry(title=name, data=user_input)
-            except (SocketGIAError, RequestException, RokuException):
+            except CannotConnect:
                 return self.async_abort(reason=ERROR_CANNOT_CONNECT)
-            except Exception:  # pylint: disable=broad-except
```

Comment:
Why are we removing this catch? We often have this in this place.

Reply:
just misunderstood what was desired :) made some changes